@sbrack8t Running this natively in Windows (not through Vagrant), this works as intended.

Running this on a PC, from inside Vagrant, I get this error:

`> gesso@0.1.0 component /vagrant/public/themes/gesso
> node component.js

/vagrant/public/themes/gesso/node_modules/inquirer/lib/prompts/list.js:32
      let index = _.findIndex(this.opt.choices.realChoices, ({ value }) => value === def);
                                                             ^

SyntaxError: Unexpected token {
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Function.promptModule.restoreDefaultPrompts (/vagrant/public/themes/gesso/node_modules/inquirer/lib/inquirer.js:55:33)
    at Object.inquirer.createPromptModule (/vagrant/public/themes/gesso/node_modules/inquirer/lib/inquirer.js:65:16)
    at Object.<anonymous> (/vagrant/public/themes/gesso/node_modules/inquirer/lib/inquirer.js:77:28)

npm ERR! Linux 2.6.32-696.6.3.el6.x86_64
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "component"
npm ERR! node v4.4.1
npm ERR! npm  v2.14.20
npm ERR! code ELIFECYCLE
npm ERR! gesso@0.1.0 component: `node component.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the gesso@0.1.0 component script 'node component.js'.
npm ERR! This is most likely a problem with the gesso package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node component.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs gesso
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!
npm ERR!     npm owner ls gesso
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /vagrant/public/themes/gesso/npm-debug.log`

--- a/component.js
+++ b/component.js
@@ -4,7 +4,7 @@ var inquirer= require('inquirer');
 init();
 
 function init() {
-  var patternsDir = "./pattern-lab/source/_patterns/";
+  var patternsDir = process.cwd() +  "/pattern-lab/source/_patterns/";
   var questions = [
     {
       type: 'input',

